### PR TITLE
Keras backend

### PR DESCRIPTION
**Pull request by Sinha-Raunak**
Task list:
WGAN_GP 
Evaluation metric
Loss specific

### CODE REVIEW

**Comment by dasguptar** [comment]
@Sinha-Raunak 
Keras needs to be added to `requirements.txt` , can you add another commit with the change?

**Comment by panwarnaveen9** [comment]
@Sinha-Raunak couple of things ? 
can you please briefly describe in pull request comment, whats are things have been added for `For keras backend` ? 

1. What you mean by keras backend. 
2. What are GANs supported ? I see, there are couple of models added by you. 
3. https://github.com/IBM/gan-toolkit/blob/74b8a9f786cb88b6c0a1d34d0890e1cebe9881fc/agant/argument_parser.py#L53 in help text you are asking query to user, just say provide the backed information and also tell 0 for pytorch and 1 keras. 
4. isn't it better to have some default value ? if its not specified by user ?
5. Please try to add more descriptive comment for commits like: *adding cgan, wgan, vanila gan for keras backend*

**Comment by panwarnaveen9** [comment]
Could you please follow best practices in following direction 

1. Add doc string comment at top of every file py file, about what that code is. Instead of one common line. 
2. Please follow this naming `module_name`, `package_name`, `ClassName`, `method_name`, `ExceptionName`, `function_name`, `GLOBAL_CONSTANT_NAME`, `global_var_name`, `instance_var_name`, `function_parameter_name`, `local_var_name ` 
following should be lower case. 
https://github.com/IBM/gan-toolkit/blob/74b8a9f786cb88b6c0a1d34d0890e1cebe9881fc/agant/make_models.py#L33
